From e10b31083dff1fcc87eb6509088d49571b487a6c Mon Sep 17 00:00:00 2001
From: Jon Medhurst <tixy@linaro.org>
Date: Wed, 6 Jun 2012 14:24:53 +0100
Subject: [PATCH] configs: Make CONFIG_MODULES part of linaro-base

CONFIG_MODULE was in both Ubuntu and Android configs so it would be
best if it is in the base config instead. Also, having
CONFIG_MODULE_UNLOAD in will enable the Gator module to be upgraded to a
later version via DKSM without rebooting, and allows for easier testing.

Signed-off-by: Jon Medhurst <tixy@linaro.org>
---
 linaro/configs/android.conf     | 2 --
 linaro/configs/linaro-base.conf | 2 ++
 linaro/configs/ubuntu.conf      | 1 -
 3 files changed, 2 insertions(+), 3 deletions(-)

diff --git a/linaro/configs/android.conf b/linaro/configs/android.conf
index 12daf61..ed2a8d8 100644
--- a/linaro/configs/android.conf
+++ b/linaro/configs/android.conf
@@ -1,5 +1,3 @@
-CONFIG_MODULES=y
-CONFIG_MODULE_UNLOAD=y
 # CONFIG_THUMB2_KERNEL is not set
 CONFIG_IPV6=y
 # CONFIG_IPV6_SIT is not set
diff --git a/linaro/configs/linaro-base.conf b/linaro/configs/linaro-base.conf
index 80bdf69..c2f23b1 100644
--- a/linaro/configs/linaro-base.conf
+++ b/linaro/configs/linaro-base.conf
@@ -11,6 +11,8 @@ CONFIG_PERF_COUNTERS=y
 CONFIG_SLAB=y
 CONFIG_PROFILING=y
 CONFIG_OPROFILE=y
+CONFIG_MODULES=y
+CONFIG_MODULE_UNLOAD=y
 CONFIG_NO_HZ=y
 CONFIG_HIGH_RES_TIMERS=y
 CONFIG_SMP=y
diff --git a/linaro/configs/ubuntu.conf b/linaro/configs/ubuntu.conf
index b8c74b8..2c6a13e 100644
--- a/linaro/configs/ubuntu.conf
+++ b/linaro/configs/ubuntu.conf
@@ -1,6 +1,5 @@
 # CONFIG_LOCALVERSION_AUTO is not set
 # CONFIG_COMPAT_BRK is not set
-CONFIG_MODULES=y
 CONFIG_DEFAULT_MMAP_MIN_ADDR=32768
 CONFIG_SECCOMP=y
 CONFIG_CC_STACKPROTECTOR=y
-- 
1.8.0

